framework/src/controller/channels/in_memory_channel.js

```diff
@@ -23,9 +23,9 @@ class InMemoryChannel extends BaseChannel {
 		super(moduleAlias, events, actions, options);
 	}
 
-	async registerToBus(bus) {
+	registerToBus(bus) {
```

Comment:
This happens same for `registerToBus` function as well. Each channel type should and must implement it `async` to have consistent usage. See the function signature in `BaseChannel`.

The top level code is expecting every channel to have async function implementation for `registerToBus` and we should keep it consistent.